### PR TITLE
Tune flash

### PR DESCRIPTION
**Pull request by MarkDaoust**
Be sure this uses flash for tuning.

### CODE REVIEW

File: quickstarts/Tuning.ipynb [hunk 1]
```diff
@@ -157,7 +157,7 @@
       },
```
Comment (pcoet) [inline quickstarts/Tuning.ipynb:157]: <pre class="ql-syntax" spellcheck="false">Line #1.    base_model = base_model = [m for m in tunable_models if 'flash' in m.name][-1]  # models/gemini-1.5-flash-001-tuning
</pre><p>Looks like a typo here: extra <code>base_model =</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

**Review by pcoet — APPROVED** [review]

**Review by pcoet — APPROVED** [review]

File: quickstarts/Tuning.ipynb [hunk 2]
```diff
@@ -282,7 +272,7 @@
         "        }\n",
         "    ],\n",
         "    id = name,\n",
-        "    epoch_count = 100,\n",
+        "    epoch_count = 10,\n",
```
Comment (MarkDaoust) [inline quickstarts/Tuning.ipynb:275]:
```suggestion
        "    epoch_count = 100,\n",
```

File: quickstarts/Tuning.ipynb [hunk 3]
```diff
@@ -187,35 +187,25 @@
...
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "w-EBSe9wTbLB"
+      },
+      "outputs": [],
+      "source": [
+        "base_model = base_model = [m for m in tunable_models if 'flash' in m.name][-1]  # models/gemini-1.5-flash-001-tuning\n",
```
Comment (MarkDaoust) [inline quickstarts/Tuning.ipynb:208]:
```suggestion
        "base_model = [m for m in tunable_models if 'flash' in m.name][-1]  # models/gemini-1.5-flash-001-tuning\n",
```

File: quickstarts/Tuning.ipynb (hunk 1 above)
Comment (MarkDaoust) [reply quickstarts/Tuning.ipynb:157]: Fixed.

**Pull request merged**